work/flow.rst

```diff
@@ -553,6 +553,15 @@ If this occurs, the merge may be reverted by anyone who notices the breakage and
 Such a fix need not use a brand new ticket and ticket branch.
 Instead, it can be performed on a "hotfix" branch named after the original ticket that was merged, typically ``tickets/DM-NNNNN-hotfix``.
 
+If a merge revert is required, the following process should be followed:
+
+- Create a new "revert" branch named ``tickets/DM-NNNNN-revert``.  This is necessary because no pushes are allowed directly to the ``main`` branch.
+- Revert the faulty merge commit on the branch.  ``git revert -m 1 {merge hash}``.  ``-m 1`` indicates that the first parent is the "mainline" code that should be kept.
```

Comment:
```suggestion
- Revert the faulty merge commit on the branch: ``git revert -m 1 {merge hash}``.  ``-m 1`` indicates that the first parent is the "mainline" code that should be kept.
```

Reply:
Can you add a line to state whether or not a PR for a revert needs to be reviewed (or if there is any other procedure to follow)?